DRUID.Rmd

```diff
@@ -7,6 +7,7 @@ author:
   url: null
 date: "`r Sys.Date()`"
 output: distill::distill_article
+    #code_folding: true
 description: |
   DRUID analysis of JHU Biobank data
 ---
```

Comment:
I can't make suggestions on lines 13-179 for some reason, might be some r markdown-github weirdness. Suggest changing: 

We used the publicly available JHU Biobank RNASeq data stored in project : syn20812185 for this analysis.

to:

We used published JHU NF Biobank RNASeq data ([view on portal](https://nf.synapse.org/Explore/Studies/DetailsPage?studyId=syn4939902)) for this analysis.

Reply:
Suggest change: femmales to females

Reply:
The DEG ggplot: possible to change to a plotly 
eg
```
p <- ggplot(data=MPNSTvsPNF_ordered....

plotly::ggplotly(p)
``` 

so it is easier to read?

Reply:
`fig.cap="Differential expression of genes in MPNST and pNF tumor types. Above shows a volcano plot highlighting significantly upregulated genes in red and downregulated genes in blue"`

That feature is so cool

Reply:
Can you make figure two smaller, particularly on the y axis?

Reply:
A couple of sentences on what figure 2 is showing would be helpful. I can kind of guess based on what I know of the labels and what I think DRUID is doing, but someone from the NF community probably wouldn't understand this figure.

Reply:
I'll file an issue, but an interesting follow up could be to look for these predictions in the cell lines/drugs tested by Field to see if any of them are "real" (they might not have been tested, of course, but drugs like palbociclib seem likely to have been tested before. 

Resveratrol as the number one hit is always a red flag to me, because it's a compound that's had a lot of hype and no real substance. more reading here if you're interested: https://blogs.sciencemag.org/pipeline/archives/2013/03/12/resveratrol_gets_some_details_cleared_up

Reply:
One question, why are there two rows and two (similar) scores for palbociclib?